custom/icds_reports/migrations/0104_citus_composite_key.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.conf import settings
+from corehq.sql_db.operations import HqRunPython
+
+
+def _citus_composite_key_migration(apps, schema_editor):
+    models_to_update = [
```

Comment:
was this list copied from the other branch? Please check it has all the right tables.

Reply:
Yeah, I think missed checking two. Updated https://github.com/dimagi/commcare-hq/pull/23544/commits/51d7c8feeecb054cc21fb1e081d0ee763e53692f